modules/ROOT/pages/deploy-resource-allocation.adoc

```diff
@@ -7,19 +7,49 @@ Before deploying a Mule application to Anypoint Runtime Fabric, determine the co
 
 When a Mule application is deployed to Runtime Fabric, the application is deployed with its own Mule runtime engine (Mule). The number of replicas, or instances of that application and runtime, are also specified. The resources available for each replica are determined by the values you set when deploying an application.
 
+The performance information provided in this topic is based on a Runtime Fabric cluster with controller nodes using AWS EC2 M4 instances. M4 instances feature a custom Intel Xeon E5-2676 v3 Haswell processor optimized specifically for EC2. They run at a base clock rate of 2.4 GHz, and can go as high as 3.0 GHz with Intel Turbo Boost. The load generator used was hosted on a separate AWS instance in the same region.
+
 You can allocate the following resources when deploying an application:
 
 * vCPU cores
+** `Reserved vCPU` - the amount of vCPU guaranteed to the application and reserved for its use.
+** `vCPU Limit` - the maximium amount of vCPU the application can use (the level to which it can burst). This is shared CPU on the worker node. 
 * Memory
 
-Performance information provided in this topic is based on a Runtime Fabric cluster with controller nodes using AWS EC2 M4 instances. M4 instances feature a custom Intel Xeon E5-2676 v3 Haswell processor optimized specifically for EC2. They run at a base clock rate of 2.4 GHz, and can go as high as 3.0 GHz with Intel Turbo Boost. The load generator used was hosted on a separate AWS instance in the same region.
+== vCPU Allocation
```

Comment:
ORG:
Be sure that this topic is introduced in the text of its container topic.
Be sure that this topic is basically parallel with and a peer of other topics at this level.
Be sure that this topic text introduces any subtopics that it contains.

Reply:
Updates applied as needed.